keyboards/fjlabs/midway60/readme.md

```diff
@@ -0,0 +1,18 @@
+# Midway60
+
+The following is the QMK Firmware for the [BolsaKeyboardSupply](https://www.bolsakeyboardsupply.com) Midway60 PCB. 
+
+The PCB will feature:
+* QMK & VIA compatibility
+* Some cool bolsa branding
+ 
+---
+
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Midway60
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader?

Reply:
Added.